Example/VimeoUpload-iOS-OldUpload/VimeoUpload-iOS-OldUpload/AppDelegate.swift

```diff
@@ -38,11 +39,11 @@ class AppDelegate: UIResponder, UIApplicationDelegate
 
         let settings = UIUserNotificationSettings(forTypes: .Alert, categories: nil)
         application.registerUserNotificationSettings(settings)
-        
-        let cameraRollViewController = CameraRollViewController(nibName: BaseCameraRollViewController.NibName, bundle:NSBundle.mainBundle())
-        let cameraNavController = UINavigationController(rootViewController: cameraRollViewController)
 
+        let cameraRollViewController = CameraRollViewController(nibName: BaseCameraRollViewController.NibName, bundle:NSBundle.mainBundle())
         let uploadsViewController = UploadsViewController(nibName: UploadsViewController.NibName, bundle:NSBundle.mainBundle())
+        
+        let cameraNavController = UINavigationController(rootViewController: cameraRollViewController)
         let uploadsNavController = UINavigationController(rootViewController: uploadsViewController)
```

Comment:
Maybe we could set the tabBarItem titles here. An item title isn't visible until its vc becomes selected.